app/infrastructure/database/migrations/002_publications_create.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+require 'sequel'
+
+Sequel.migration do
+  change do
+    create_table(:publications) do
+      primary_key :id
+      # 感覺錯錯的
+      # foreign_key  :publicationId, :papers
+      
+      Integer      :publicationId, unique: true, null: false
```

Comment:
this line seems duplicated with primary_key.

Reply:
perhaps `publicationId` is a key from their 3rd party API?